src/gui/attributetable/qgsdualview.cpp

```diff
@@ -912,9 +916,11 @@ void QgsDualView::setFeatureSelectionManager( QgsIFeatureSelectionManager *featu
 void QgsDualView::setAttributeTableConfig( const QgsAttributeTableConfig &config )
 {
   mConfig = config;
-  mLayer->setAttributeTableConfig( config );
-  mFilterModel->setAttributeTableConfig( config );
-  mTableView->setAttributeTableConfig( config );
+  if ( mLayer )
```

Comment:
Do we need this check?
It only guards the first access to layer. I guess if there is never a crash with line 921 we can also safely assume that layer is present for 920.

Reply:
Good point. Coverity would not be happy!